File: projects_manager_backend/app/Models/Project.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Project extends Model
{
    use HasFactory;


    //
    protected $primaryKey = 'project_id' ;


    //
    public $timestamps = false;


    //
    protected $fillable = [
        'title' ,
        'about' 
    ] ;


        

}

File: Mobile/projects_manager/lib/data/models/project.dart
//
//
//

class Project {
  int? projectId;
  String? title;
  String? about;

  Project({
    this.projectId ,
    this.title,
    this.about
});

  Project.fromJson(Map<String, dynamic> json) {
    projectId = json["project_id"];
    title = json['title'];
    about = json["about"];
  }
}
